### PR TITLE
Add an image field to BuyersGuideProductCategory

### PR DESCRIPTION
**Pull request by mmmavis**
Related PRs/issues #5537

front end / template work will be tackled in the next PR

### CODE REVIEW

File: network-api/networkapi/buyersguide/pagemodels/get_category_og_image_upload_path.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from networkapi.utility.images import get_image_upload_path
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/pagemodels/get_category_og_image_upload_path.py:1]: note that this is not a page model, but a helper function, let's put it in utils.py

File: network-api/networkapi/buyersguide/pagemodels/product_category.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
 from django.db import models
 from django.utils.text import slugify
 
-
 from wagtail.snippets.models import register_snippet
 
 from .products.base import Product
+from .get_category_og_image_upload_path import get_category_og_image_upload_path
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/pagemodels/product_category.py:7]:
```suggestion
from ..utils import get_category_og_image_upload_path
```

**Review by Pomax — CHANGES_REQUESTED** [review]
small change

File: network-api/networkapi/buyersguide/pagemodels/get_category_og_image_upload_path.py (hunk 1 above)
Comment (mmmavis) [reply network-api/networkapi/buyersguide/pagemodels/get_category_og_image_upload_path.py]: yea totally makes sense!

**Review by Pomax — CHANGES_REQUESTED** [review]

**Comment by mmmavis** [comment]
Re-created the migration file as well as the review app.

**Review by Pomax — APPROVED** [review]
no issues migrating anymore!

**Pull request merged**